fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/FabricRegistryInit.java

```diff
@@ -212,5 +215,13 @@ public void onInitialize() {
 		// Synced by rawID.
 		RegistryAttributeHolder.get(Registries.MAP_DECORATION_TYPE)
 				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Synced via PacketCodecs.registry
+		RegistryAttributeHolder.get(Registries.ARMOR_MATERIAL)
+				.addAttribute(RegistryAttribute.SYNCED);
+
+		// Synced via PacketCodecs.registry
+		RegistryAttributeHolder.get(Registries.ARMOR_MATERIAL)
```

Comment:
duplicate?

Reply:
Oh yeah :D Not sure how I managed that, blame my terrible keyboard I guess